spector/frontend/src/routes/profile.js

```diff
@@ -21,12 +20,9 @@ const Profile = () => {
       <h2 className="ProfileTittle"> Profile Page</h2>
         <ul className = "ba">
           {data.map((item, index) => (
-            <li key={index}>
```

Comment:
You're mapping `data` which contains Name, Bio and Age. You can't pass these items to `<ActivityPopup />`

1. Merge main into this branch (to get the changes Val made where activities are included in profile - this means `data` will now have Name, Bio, Age and activities)

3. Display Name, Bio and Age like you were doing before
4. To display activities map `data.activities` and pass those items to `<ActivityPopup />`